config.inc.php

```diff
@@ -802,6 +802,8 @@ function x_send_sms_clickatell($to, $message) {
 
 $CONF['version'] = '3.4-dev';
 
+$CONF['smtp_active_flag'] = 'NO';
```

Comment:
this needs a line or two of docs to explain what the flag is for - perhaps linking to the DOVECOT.txt file etc?

Reply:
Thanks for the feedback. I've added a description and link to DOVECOT.txt